nrfu_tests/test_interfaces_mlag_status_check.py

```diff
@@ -0,0 +1,95 @@
+# Copyright (c) 2022 Arista Networks, Inc.  All rights reserved.
+# Arista Networks, Inc. Confidential and Proprietary.
+
+"""
+Test cases for verification of system hardware free memory functionality
+"""
+
+import pytest
+from pyeapi.eapilib import EapiError
+from vane.logger import logger
+from vane.config import dut_objs, test_defs
+from vane import tests_tools
+
+TEST_SUITE = "nrfu_tests"
+
+
+@pytest.mark.nrfu_tests
+@pytest.mark.interfaces
+class InterfacesMlagStatusCheckTests:
+    """
+    Testcase for verification of mlag interfaces status check.
+    """
+
+    dut_parameters = tests_tools.parametrize_duts(TEST_SUITE, test_defs, dut_objs)
+    test_duts = dut_parameters["test_interfaces_mlag_status_check"]["duts"]
+    test_ids = dut_parameters["test_interfaces_mlag_status_check"]["ids"]
+
+    @pytest.mark.parametrize("dut", test_duts, ids=test_ids)
+    def test_interfaces_mlag_status_check(self, dut, tests_definitions):
+        """
+        TD: Testcase for verification of mlag interfaces status check.
+        Args:
+            dut(dict): details related to a particular DUT
+            tests_definitions(dict): test suite and test case parameters
+        """
+        tops = tests_tools.TestOps(tests_definitions, TEST_SUITE, dut)
+        self.output = ""
+        tops.actual_output = {"mlag_details": {}}
+        tops.expected_output = {"mlag_details": {}}
+
+        # Forming output message if test result is passed
+        tops.output_msg = "The mlag status is displayed."

```

Comment:
```suggestion
        tops.output_msg = "MLAG is configured on the device."

```
Update at all relevant places